File: fizzbuzz.py
def fizzbuzz(number):
    # if type(number) != int:
    #     raise TypeError("Incorrect input type, should be integer")
    if number % 15 == 0:
        answer = "FizzBuzz"
    elif number % 3 == 0:
        answer = "Fizz"
    elif number % 5 == 0:
        answer = "Buzz"
    else: answer = number
    
    if type(answer) != int:
        result = answer
    else:
        result = number
    
    return result
    # return msg

File: fizzbuzz.js
function shoutNumber(numberShouted) {
  if (numberShouted % 15 === 0) {
    return "FizzBuzz";
  } else if (numberShouted % 3 === 0) {
    return "Fizz";
  } else if (numberShouted % 5 === 0) {
    return "Buzz";
  }
  return numberShouted;
}

for (let counter = 0; counter < 100; counter++) {
  shoutNumber(counter);
}

export { shoutNumber };
